frontend/tests/unit/service/comsService.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { comsService } from '@/services';
+import { comsAxios } from '@/services/interceptors';
+
+vi.mock('vue-router', () => ({
+  useRouter: () => ({
+    push: vi.fn(),
+    replace: vi.fn()
+  })
+}));
+
+const PATH = '/object';
+const deleteSpy = vi.fn();
+const getSpy = vi.fn();
+const putSpy = vi.fn();
+const windowSpy = vi.spyOn(window, 'open').mockImplementation(() => vi.fn() as any);
```

Comment:
It's generally better to only use any context specific mockImplementations in either the body of a lifecycle manager (like beforeEach) or on each specific test; doing it chained on the declaration is very brittle as if things are cleared/reset, the mockImplementation would no longer apply and potentially bust later tests.